cfgov/jinja2/v1/_includes/molecules/card.html

```diff
@@ -1,4 +1,17 @@
+{# ==========================================================================
+
+   Card
+
+   ==========================================================================
+
+   Description:
+
+   Creates a card with header and body text.
+
+   ========================================================================== #}
+
+{# TODO: This is a prototype and needs to change the text via parameters. #}
 <div class="m-card">
-  <h3 class="h1 m-card_heading">Card Heading</h3>
-  <p class="m-card_body">Card text</p>
+    <h3 class="m-card_heading h1">Card Heading</h3>
```

Comment:
The `h1` class would be imported from Capital Framework first, so our molecule would in theory overwrite that in the cascade. Order in markup doesn't matter, but I always prefer keeping them the same so it's easier to track down bugs.